cpu/nrf5x_common/clock.c

```diff
@@ -39,14 +43,34 @@ void clock_init_hf(void)
 #endif
 #endif
 
-#if CLOCK_HFCLK
-    /* start the HF clock */
-    NRF_CLOCK->EVENTS_HFCLKSTARTED = 0;
-    NRF_CLOCK->TASKS_HFCLKSTART = 1;
-    while (NRF_CLOCK->EVENTS_HFCLKSTARTED == 0) {}
+    /* allow to always enable the HFXO as non-default option */
+#if CLOCK_HFXO_ONBOOT
```

Comment:
This option is never defined / documented.

Do we need it at all?

Reply:
On paper no, but there might be cases, e.g. for debugging, where this is useful. But you are right, I should document it somewhere :-)